packages/modules/src/lib/replaceArticleCount.tsx

```diff
@@ -37,3 +37,16 @@ export const replaceArticleCount = (
 
     return elements;
 };
+
+export const replaceArticleCount = (
```

Comment:
previously this function would always make the article count a link (for the opt out). But we do not want to do this if the 'above' article count is also displayed.
So with this PR we pass in `optOutLink: boolean`